sdk/src/main/java/me/uport/sdk/Uport.kt

```diff
@@ -163,11 +141,41 @@ object Uport {
         }
 
         runBlocking { accountCreator.deleteAccount(rootHandle) }
-        if (rootHandle == defaultAccount?.handle) {
-            defaultAccount = null
-        }
     }
 
     fun deleteAccount(acc: Account) = deleteAccount(acc.handle)
 
-}
+    internal fun migrateAccounts(oldPrefs: SharedPreferences, accountStorage: AccountStorage) {
```

Comment:
It is very important to call the migration method during initialization.
There's no reliable way for devs to find out they need to call this.

Reply:
I forgot to add this to the initialization